### DISCUSSION TITLE
Stryker.Net, which unittests have killed mutants?

### DISCUSSION THREAD

**Discussion by JustineSondore** [stryker-mutator/stryker-net#2459] [General]
Hi, I'm using Stryker.Net Version 3.7.1; what is the best way to tell if a unittest has actually killed any mutants? I've found that if `--verbosity trace` is used, then the output contains info like this:
`Test run for 123: A kind of mutation is failed with output: NameOfTheTestThatFailed` , followed by the test output.
Is there a way to get a report/overview?

**Comment by rouke-broersma** [comment]
The start for these kinds of insights is coming with #1850 and should be enhanced by #1784

**Reply by JustineSondore** [reply]
Good to know, thanks! :)
